packages/client/components/TeamHealth.tsx

```diff
@@ -49,26 +95,66 @@ const TeamHealth = (props: Props) => {
           <PhaseHeaderTitle>{phaseLabelLookup.TEAM_HEALTH}</PhaseHeaderTitle>
         </MeetingTopBar>
         <PhaseWrapper>
-          <div className='text-center text-2xl'>{question}</div>
-          <form
-            className='flex flex-row'
-            onChange={(e: React.ChangeEvent<HTMLFormElement>) =>
-              console.log('TODO: Vote changed', e.target.value)
-            }
-          >
-            {labels?.map((label) => (
-              //center vertically
-              <div key={label} className='flex h-24 w-24 items-center justify-center p-8'>
-                <input name='foo' className='peer hidden' type='radio' id={label} value={label} />
-                <label
-                  htmlFor={label}
-                  className='text-4xl opacity-75 drop-shadow-lg hover:text-5xl hover:opacity-100 hover:blur-none peer-checked:text-6xl peer-checked:opacity-100 peer-checked:blur-none'
+          <div className='flex h-[300px] flex-col items-center'>
```

Comment:
+1 when hovering over the card, it would be nice if the cursor changed to a `pointer` to indicate that the card is clickable